cloudbuild/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# TODO(ekrekr): update to image `gcr.io/cloud-builders/bazel` once we're at Bazel version 7+.
+FROM gcr.io/cloud-builders/bazel:5.4.0
```

Comment:
```
FROM index.docker.io/debian:bullseye
```

There's no point pulling from bazel as you install bazelisk below anyway

Reply:
I realised we can get rid of the docker publish and run step entirely! see new script.